truss/templates/shared/log_config.py

```diff
@@ -0,0 +1,122 @@
+import logging
+import os
+import urllib.parse
+from typing import Any, Mapping
+
+from pythonjsonlogger import jsonlogger
+
+LOCAL_DATE_FORMAT = "%H:%M:%S"
+
+
+def _disable_json_logging() -> bool:
+    return bool(os.environ.get("DISABLE_JSON_LOGGING"))
+
+
+class _HealthCheckFilter(logging.Filter):
+    def filter(self, record: logging.LogRecord) -> bool:
+        excluded_paths = {
+            "GET / ",
+            "GET /v1/models/model ",
+            "GET /v1/models/model/loaded ",
+        }
+        msg = record.getMessage()
+        return not any(path in msg for path in excluded_paths)
+
+
+class _AccessJsonFormatter(jsonlogger.JsonFormatter):
+    def format(self, record: logging.LogRecord) -> str:
+        # Uvicorn sets record.msg = '%s - "%s %s HTTP/%s" %d' and
+        # record.args = (addr, method, path, version, status).
+        # Python's logging system resolves final
+        # record.message = record.msg % record.args unless we override record.msg.
+        if record.name == "uvicorn.access" and record.args and len(record.args) == 5:
+            client_addr, method, raw_path, version, status = record.args
+            path_decoded = urllib.parse.unquote(str(raw_path))
+            new_message = (
+                f"Handled request from {client_addr} - {method} "
+                f"{path_decoded} HTTP/{version} {status}"
+            )
+            record.msg = new_message
+            record.args = ()  # Ensure Python doesn't reapply the old format string
+        return super().format(record)
+
+
+class _AccessFormatter(logging.Formatter):
+    def format(self, record: logging.LogRecord) -> str:
+        if record.name == "uvicorn.access" and record.args and len(record.args) == 5:
+            client_addr, method, raw_path, version, status = record.args
+            path_decoded = urllib.parse.unquote(str(raw_path))
+            new_message = f"Incoming {client_addr} - {method} {path_decoded} HTTP/{version} {status}"
```

Comment:
Why did we need to make this change?  I think if we're touching this at all we should probably exclude the IP Address either, that's been there for a long time and is not meaningful to userse

Reply:
yeah, these are internal IPs so I think we should drop them (fine to do in follow-up). Trace IDs are at least useful to us, so makes sense to include